### ISSUE TITLE
StackBlitz Examples Broken

### ISSUE THREAD

**Issue by andrew-frueh** [HealthCatalyst/Fabric.Cashmere#1243]
**Summary**

After some recent CSS updates, our StackBlitz examples have stopped working (see screenshot):

![Screen Shot 2020-04-30 at 1 56 47 PM](https://user-images.githubusercontent.com/22795893/80753568-b57bef00-8aea-11ea-9f8a-c66ae6c93d5c.png)

**Comment by joeskeen** [comment]
@andrew-frueh looks like it may be broken again, but only when running locally - I just tried StackBlitz on my PR branch locally and all the examples do the `/dev/null` error.  Prod site seems to work.  Did you learn anything in fixing this that may be useful in fixing this locally?

**Comment by andrew-frueh** [comment]
@joeskeen - the fix I had to make last time had to with the scss import paths not working on StackBlitz.  I actually didn't think StackBlitz _could_ work locally.  I'm pretty sure I had to make my fixes over on StackBlitz, write them into the library, and cross my fingers and push a build.  Then I verified after the build was published.  So my guess is that local testing has probably been broken for a while.
